stdlib/Distributed/src/clusterserialize.jl

```diff
@@ -173,9 +173,9 @@ function deserialize_global_from_main(s::ClusterSerializer, sym)
     sym_isconst = deserialize(s)
     v = deserialize(s)
     if sym_isconst
-        @eval Main const $sym = $v
+        ccall(:jl_set_const, Cvoid, (Any, Any, Any), Main, sym, v)
     else
-        @eval Main $sym = $v
+        ccall(:jl_set_global, Cvoid, (Any, Any, Any), Main, sym, v)
     end
 end
```

Comment:
This changes the implicit value of the function. Does that matter? Should we still have `return v` at the end?

Reply:
That's a good point. I believe the value is unused though, so I'll add an explicit `return nothing`.